src/Components/Web.JS/src/Boot.Web.ts

```diff
@@ -30,6 +31,14 @@ function boot(options?: Partial<WebStartOptions>) : Promise<void> {
   started = true;
 
   Blazor._internal.loadWebAssemblyQuicklyTimeout = 3000;
+  // Definedh ere to avoid inadvertently imported enhanced navigation
+  // related APIs in WebAssembly or Blazor Server contexts.
```

Comment:
Nit:

```suggestion
  // Defined here to avoid inadvertently imported enhanced navigation
  // related APIs in WebAssembly or Blazor Server contexts.
```